src/OrleansAWSUtils/Membership/DynamoDBGatewayListProvider.cs

```diff
@@ -45,9 +45,15 @@ public Task InitializeGatewayListProvider(ClientConfiguration conf, Logger logge
             var expressionValues = new Dictionary<string, AttributeValue>
             {
                 { $":{SiloInstanceRecord.DEPLOYMENT_ID_PROPERTY_NAME}", new AttributeValue(deploymentId) },
-                { $":{SiloInstanceRecord.STATUS_PROPERTY_NAME}", new AttributeValue { N =  INSTANCE_STATUS_ACTIVE } }
+                { $":{SiloInstanceRecord.STATUS_PROPERTY_NAME}", new AttributeValue { N = INSTANCE_STATUS_ACTIVE } },
+                { $":{SiloInstanceRecord.PROXY_PORT_PROPERTY_NAME}", new AttributeValue { N = "0"} }
```

Comment:
@sergeybykov this only means we are setting the attribute as a Number to 0 so it can be used later in the real query. It is not the query itself.